dhcpv6/server_test.go

```diff
@@ -9,21 +9,36 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// utility function to return the loopback interface name
+func getLoopbackInterface() string {
+        var ifaces []net.Interface
+        var err error
+        if ifaces, err = net.Interfaces(); err != nil {
```

Comment:
I prefer to return an error here, so it is obvious what happened, i.e. missing loopback interface? Or any other error?

Reply:
Added. :)